code/script.js

```diff
@@ -205,7 +257,7 @@ let secret
 let currentQuestion
 let charactersInPlay
 
```

Comment:
Seems to be working, so maybe no need to fix it, but most code I've seen has had the secret defined outside of the function, as a global variable.